spyderlib/plugins/shortcuts.py

```diff
@@ -415,14 +417,17 @@ def save(self):
         set_shortcut(self.context, self.name, self.key)
 
 
-CONTEXT, NAME, SEQUENCE = [0, 1, 2]
+CONTEXT, NAME, SEQUENCE, SEARCH_SCORE = [0, 1, 2, 3]

```

Comment:
I'm getting this message when trying to filter shortcuts:

```python-traceback
Traceback (most recent call last):
  File "/home/carlos/Projects/spyder/github-repo/spyderlib/plugins/shortcuts.py", line 698, in set_regex
    self.sortByColumn(SEARCH_SCORE)
TypeError: QTableView.sortByColumn(int, Qt.SortOrder): not enough arguments
```

and I think that's because we have three columns in the shortcuts table, and so SEARCH_NAME is not a valid column number.